devstack_romana.md

```diff
@@ -0,0 +1,93 @@
+```sh-session
+ubuntu@ip-192-168-99-10:~$ nova boot --flavor m1.nano --image cirros-0.3.4-x86_64-uec --key-name shared-key --nic net-id=$(neutron net-show romana -Fid -f value) --meta romanaSegment=default example-instance
++--------------------------------------+----------------------------------------------------------------+
+| Property                             | Value                                                          |
++--------------------------------------+----------------------------------------------------------------+
+| ...                                  |                                                                |
+| created                              | 2016-03-02T22:46:38Z                                           |
+| flavor                               | m1.nano (42)                                                   |
+| ...                                  |                                                                |
+| id                                   | 048054fe-ec19-45ff-a679-83c19ebb5026                           |
+| image                                | cirros-0.3.4-x86_64-uec (c11f6cde-7bb8-40ed-bce8-fedf676ef1ff) |
+| ...                                  |                                                                |
+| metadata                             | {"romanaSegment": "default"}                                   |
+| name                                 | example-instance                                               |
+| ...                                  |                                                                |
++--------------------------------------+----------------------------------------------------------------+
+ubuntu@ip-192-168-99-10:~$ nova list
++--------------------------------------+------------------+--------+------------+-------------+------------------+
+| ID                                   | Name             | Status | Task State | Power State | Networks         |
++--------------------------------------+------------------+--------+------------+-------------+------------------+
+| 048054fe-ec19-45ff-a679-83c19ebb5026 | example-instance | ACTIVE | -          | Running     | romana=10.0.17.3 |
++--------------------------------------+------------------+--------+------------+-------------+------------------+
+
+```
+
+A new instance was created, and has IP `10.0.17.3`. You can SSH into this from the host it was created on.
+
+```
+ubuntu@ip-192-168-99-10:~$ ssh cirros@10.0.17.3
+The authenticity of host '10.0.17.3 (10.0.17.3)' can't be established.
+RSA key fingerprint is f6:2a:53:3c:e6:62:62:52:6b:17:ba:53:23:1f:c8:3d.
+Are you sure you want to continue connecting (yes/no)? yes
+Warning: Permanently added '10.0.17.3' (RSA) to the list of known hosts.
+$ 
+```
+
+## Details
+
+- See what `iptables` were configured: `sudo iptables -nL`
```

Comment:
I think that belongs in a separate, comprehensive page and for this bit, just an "ooh, automagic iptables rules. nice" is what we're aiming for.